extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -97,6 +137,49 @@ const MOTION_DURATION_RATIO = 0.8; // fraction of animation
  */
 let LightboxElementMetadataDef;
 
+/**
```

Comment:
We should probably have a utility library for these functions. IIRC they're definitely used in more than just here.

Reply:
Ok, created #21104, will try to go through a few extensions to see what other things pop up as well.